### PR TITLE
minify code

### PR DESCRIPTION
**Pull request by moritzwilksch**
storage is becoming increasingly expensive:

To save on storage, let's use a code minifier to make the python codebase nice and lean.

### CODE REVIEW

**Review by pavelzw — CHANGES_REQUESTED** [review]
Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://tinyurl.com/cla-tool-helper-utils) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.

---

I have read the CLA Document and I hereby sign the CLA

**Comment by moritzwilksch** [comment]
I have read the CLA Document and I hereby sign the CLA

image

File: tool_helper_utils/__init__.py [hunk 1]
```diff
@@ -1,41 +1,14 @@
...
-    def find_spec(self, fullname, path=None, target=None):
-        return self.find_module(fullname)
-
-
-sys.meta_path.insert(0, StopItImportHook())
+	def get_bad_idea(A):return'factory'
+class StopItImportHook(importlib.abc.MetaPathFinder,importlib.abc.Loader):
+	BAD_IDEAS='tool','helper','util',FactoryFactory(FactoryBadIdea()).build(),'builder'
+	def find_module(A,fullname,path=None):
+		if any(A in fullname.lower()for A in A.BAD_IDEAS):raise ImportError('stop it')
```
Comment (pavelzw) [inline tool_helper_utils/__init__.py:12]: great idea to use single quotes instead of double quotes! I would guess they take up half the space 🚀 📈

**Review by pavelzw — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  pre-commit-checks:
+    name: Pre-commit Checks
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659
```
Comment (pavelzw) [inline .github/workflows/ci.yaml]:
```suggestion
        uses: prefix-dev/setup-pixi@v0
```

this should save some bytes as well

Comment (moritzwilksch) [reply .github/workflows/ci.yaml]: thank you for your valuable feedback! fixed in https://github.com/cmdevries/ToolHelperUtils/pull/3/commits/0000000e3e98a924131bd6d0f44fc666b804335b

**Comment by cmdevries** [comment]
How about using the C preprocessor to implement dictionary based compression?

**Pull request merged**